src/schemas/const.js

```diff
@@ -0,0 +1,3 @@
+export const REQUIRED_ERROR = 'This field is required.';
```

Comment:
Errors aren't connected with schemas. Why? Cause they can appear even when you dont use them so more proper directory would be 'src/constants' and file 'errors.ts'

Reply:
Second avoid using file names shortcuts jak 'const'. Some day you will meet a situation when you will have name clash because of that or the shortcut won't be verbose enough. It lowers code readability